habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/PyenvAndPoetrySetup.java

```diff
@@ -191,6 +191,17 @@ private String validatateAndConfigurePyenv(List<String> missingRequiredToolMsgs,
                 currentPythonVersion = pyenvHelper.getCurrentPythonVersion();
             }
 
```

Comment:
I: this is the meat of the change - when pyenv isn't configured correctly but we have configured habushu to use it, pyenv says your python version is one thing, but python calls on the path say another.